_pages/en/software-assessment-sharing-pilot.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: default
+title: Software Assessment Sharing Pilot
+ref: software-assessment-sharing-pilot
+lang: en
+status: In Progress;primary
+sections: Pilots
+permalink: /software-assessment-sharing-pilot.html
+---
+
+- [Problem Statement](#problem-statement)
+- [Expected Results](#expected-results)
+- [GCxchange Group](#gcxchange-group)
+- [Sharing ESDC Software Assessments](#sharing-esdc-software-assessments)
+- [Feedback](#feedback)
+- [Next Steps](#next-steps)
+
+The idea of sharing software assessments documents and results between departments has been discussed many times on GCconnex over the years but was never formally put in place.
+Some files were shared but they are not easy to find or were only shared through email.
+GCxchange is a new platform available to all GC departments and agencies, that have onboarded to M365.
+In time, it should replace GCconnex/GCmessage as the interdepartmental collaboration space for GC employees.
+The IT Strategy team at ESDC put in place a document repository on GCxchange and confirmed that the GC is one legal entity, hence sharing software assessments between departments is not a problem.
+This is a one-year pilot at ESDC and we plan to keep sharing our software assessments until August 2023.
+
+## Problem Statement
+
+GC departments are individually assessing and certifying their software assets.
+This way of working leads to many issues, including:
```

Comment:
remove "many" to just say
"This way of working leads to the following issues: "